netsuite/10.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc

```diff
@@ -10,96 +10,114 @@ Upgrade NetSuite Connector to version 10.0 to use the new features.
 
 [%header%autowidth.spread]
 |===
 |Source name |Old XML name | New XML name
 
-| On New Object | `<netsuite:new-object-trigger ...` | `<netsuite:new-object-listener ...`
-| On Modified Object |  `<netsuite:modified-trigger ...` | `<netsuite:modified-listener ...`
-| On Deleted Object | `<netsuite:deleted-trigger ...` | `<netsuite:deleted-listener ...`
+| On New Object | `<netsuite:new-object-trigger` | `<netsuite:new-object-listener`
+| On Modified Object |  `<netsuite:modified-trigger` | `<netsuite:modified-listener`
+| On Deleted Object | `<netsuite:deleted-trigger` | `<netsuite:deleted-listener`
 |===
 
 == Upgrade Steps
 
-If you are upgrading from an earlier version of the connector, a small popup appears in the bottom right corner of Anypoint Studio with the `Updates Available` message.
+If you are upgrading from an earlier version of the connector:
 
-Click the pop-up and follow the prompts to install the latest version.
+. A small pop-up message appears in the bottom right corner of Anypoint Studio with the `Updates Available` message.
+. Click the pop-up message and follow the prompts to install the latest version.
 
 You can also follow these steps to perform the upgrade:
 
 . In Anypoint Studio, create a Mule project.
 . In *Mule Palette*, click *Search in Exchange*.
-. In *Add Dependencies to Project*, enter `NetSuite` in the search field.
-. In *Available modules*, select *NetSuite* and click *Add*.
-. Click *Finish*.
-. Verify that the NetSuite Connector dependency version is 10.0.0 in the `pom.xml` file in the Mule project.
-
+. In *Add Dependencies to Project*, enter `netsuite` in the search field.
+. In *Available modules*, select *NetSuite* and click *Add*. +
 Studio upgrades the connector automatically.
+. Click *Finish*.
+. In the `pom.xml` file for the Mule project, verify that the NetSuite Connector dependency version is `10.0.0`. 
 
-== Additional Notes
+== Additional Tasks
 
 * If you were using one of the authentication schemas that were removed, you must change your configurations so that they use either *Request Based* or *Token* authentication. 
 * You must verify that your error handling is updated and works as expected, given the minor changes in error codes in this version of the connector. 
-* Refer to the previous table for updated operations and sources. If your integration uses any of the affected components, make the necessary changes to comply with the new interfaces.
+* Refer to <<changed_operations,Changed Operations>> and <<changed_sources,Changed Sources>> for updated operations and sources. +
+If your integration uses any of the affected operations and sources, make the necessary changes to comply with the new interfaces.
 * All DataWeave mappings that create objects for operation inputs must be updated, since the package of the exposed classes was changed to `com.mulesoft.connector.netsuite.extension.api`. +
 You must also update mappings that transform object outputs.
```

Comment:
This item is NOT covered earlier and perhaps should have its own subtopic, as do the others.

Reply:
BREAK HERE, SECOND EDIT

Reply:
RESUME SECOND EDIT HERE